cmd/kubermatic-api/options.go

```diff
@@ -90,7 +90,7 @@ func newServerRunOptions() (serverRunOptions, error) {
 	flag.StringVar(&s.updatesFile, "updates", "updates.yaml", "The updates.yaml file path")
 	flag.StringVar(&s.presetsFile, "presets", "", "The optional file path for a file containing presets")
 	flag.StringVar(&s.swaggerFile, "swagger", "./cmd/kubermatic-api/swagger.json", "The swagger.json file path")
-	flag.StringVar(&rawAccessibleAddons, "accessible-addons", "", "Comma-separated list of user cluster addons to expose via the API")
+	flag.StringVar(&rawAccessibleAddons, "accessible-addons", "multus", "Comma-separated list of user cluster addons to expose via the API")
```

Comment:
What makes multus so special, compared to the other default accessible addons?

Reply:
It's not special at all. I will remove it but it wouldn't make big difference I think since operator has its own config. It would matter only if someone would run it without any config and the defaults would be used.